### ISSUE TITLE
Clicking the AudioBand toolbar shuffles between profiles

### ISSUE THREAD

**Issue by sorsted** [AudioBand/AudioBand#101]
## Description
Whenever I click the AudioBand toolbar (in any location that is not a button), AudioBand changes profile. Clicking it repeatedly shuffles between the different profiles. I tried uninstalling and installing again, but the problem persists.

## How To Reproduce
1. Click in any place on the AudioBand toolbar outside the buttons.
2. Observe how it shuffles between profiles.
3. Get really annoyed because you have to right click and find the correct profile again.

## Version
 - Windows 10 Home 21H2
 - AudioBand v1.2.1

**Comment by TechSupport26** [comment]
Are you using any custom mousebindings?

**Comment by sorsted** [comment]
No, no custom mouse bindings here.

One thing I might mention: In the previous version of AudioBand (the first one with “Like button” support), I made a new profile and later tried to delete it. The deletion didn’t work, but the profile started to behave weirdly. Then after the update to v1.2.1, the profile worked again. But it could be that some data was corrupted.

**Comment by TechSupport26** [comment]
Can you upload AudioBand's log file in %temp%

**Comment by sorsted** [comment]
I have four log files, one called simply `AudioBand.log`, and others called `AudioBand.<some date>.log`, which one(s) do you want? And more importantly, how do I upload in here? Should I simply write them here in the comments.

**Comment by TechSupport26** [comment]
The latest one will do.
You can copy-paste it into the comment. Or you can share it a different way if you don't want your home file name to be visible for everyone.

**Comment by sorsted** [comment]
Here you go:
```
2023-02-02 09:49:48.6827 | INFO | AudioSource(Spotify) | Tried to update Spotify Playback, but user has no internet connection. Update Interval is 30 seconds.
2023-02-02 09:50:14.5979 | DEBUG | AudioSource(Spotify) | Using RefreshToken from previous auth to connect to Spotify.
2023-02-02 09:50:14.7844 | DEBUG | AudioSource(Spotify) | Received new access token. Expires in: 01:00:00 (At 02/02/2023 10.50.14)
2023-02-02 13:54:41.9348 | INFO | AudioSource(Spotify) | Tried to update Spotify Playback, but user has no internet connection. Update Interval is 30 seconds.
2023-02-02 13:55:12.0370 | INFO | AudioSource(Spotify) | Tried to update Spotify Playback, but user has no internet connection. Update Interval is 30 seconds.
2023-02-02 13:55:42.1896 | DEBUG | AudioSource(Spotify) | Using RefreshToken from previous auth to connect to Spotify.
2023-02-02 13:55:42.4100 | DEBUG | AudioSource(Spotify) | Received new access token. Expires in: 01:00:00 (At 02/02/2023 14.55.42)
2023-02-02 14:55:45.6000 | DEBUG | AudioSource(Spotify) | Using RefreshToken from previous auth to connect to Spotify.
2023-02-02 14:55:45.8219 | DEBUG | AudioSource(Spotify) | Received new access token. Expires in: 01:00:00 (At 02/02/2023 15.55.45)
2023-02-02 15:55:46.3464 | DEBUG | AudioSource(Spotify) | Using RefreshToken from previous auth to connect to Spotify.
2023-02-02 15:55:46.5917 | DEBUG | AudioSource(Spotify) | Received new access token. Expires in: 01:00:00 (At 02/02/2023 16.55.46)
2023-02-02 16:55:47.2607 | DEBUG | AudioSource(Spotify) | Using RefreshToken from previous auth to connect to Spotify.
2023-02-02 16:55:47.5297 | DEBUG | AudioSource(Spotify) | Received new access token. Expires in: 01:00:00 (At 02/02/2023 17.55.47)
2023-02-02 17:55:50.7550 | DEBUG | AudioSource(Spotify) | Using RefreshToken from previous auth to connect to Spotify.
2023-02-02 17:55:50.9923 | DEBUG | AudioSource(Spotify) | Received new access token. Expires in: 01:00:00 (At 02/02/2023 18.55.50)
2023-02-02 19:19:53.5612 | INFO | AudioSource(Spotify) | Tried to update Spotify Playback, but user has no internet connection. Update Interval is 30 seconds.
2023-02-02 19:20:15.4970 | DEBUG | AudioSource(Spotify) | Using RefreshToken from previous auth to connect to Spotify.
2023-02-02 19:20:15.7834 | DEBUG | AudioSource(Spotify) | Received new access token. Expires in: 01:00:00 (At 02/02/2023 20.20.15)
2023-02-02 20:20:16.1309 | ERROR | AudioSource(Spotify) | SpotifyAPI.Web.APIUnauthorizedException: The access token expired
   at SpotifyAPI.Web.Http.APIConnector.ProcessErrors(IResponse response)
   at SpotifyAPI.Web.Http.APIConnector.<DoRequest>d__31.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at SpotifyAPI.Web.Http.APIConnector.<DoSerializedRequest>d__30`1.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at SpotifyAPI.Web.Http.APIConnector.<SendAPIRequest>d__34`1.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at SpotifyAudioSource.SpotifyAudioSource.<NotifyLike>d__123.MoveNext() in C:\Users\vanro\Documents\Programming\AudioBand\AudioBand\src\SpotifyAudioSource\SpotifyAudioSource.cs:line 776
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at SpotifyAudioSource.SpotifyAudioSource.<UpdatePlayer>d__128.MoveNext() in C:\Users\vanro\Documents\Programming\AudioBand\AudioBand\src\SpotifyAudioSource\SpotifyAudioSource.cs:line 904---START OF EXCEPTION---
SpotifyAPI.Web.APIUnauthorizedException: The access token expired
   at SpotifyAPI.Web.Http.APIConnector.ProcessErrors(IResponse response)
   at SpotifyAPI.Web.Http.APIConnector.<DoRequest>d__31.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at SpotifyAPI.Web.Http.APIConnector.<DoSerializedRequest>d__30`1.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at SpotifyAPI.Web.Http.APIConnector.<SendAPIRequest>d__34`1.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at SpotifyAudioSource.SpotifyAudioSource.<NotifyLike>d__123.MoveNext() in C:\Users\vanro\Documents\Programming\AudioBand\AudioBand\src\SpotifyAudioSource\SpotifyAudioSource.cs:line 776
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at SpotifyAudioSource.SpotifyAudioSource.<UpdatePlayer>d__128.MoveNext() in C:\Users\vanro\Documents\Programming\AudioBand\AudioBand\src\SpotifyAudioSource\SpotifyAudioSource.cs:line 904
---END OF EXCEPTION

2023-02-02 20:20:19.4216 | DEBUG | AudioSource(Spotify) | Using RefreshToken from previous auth to connect to Spotify.
2023-02-02 20:20:19.6100 | DEBUG | AudioSource(Spotify) | Received new access token. Expires in: 01:00:00 (At 02/02/2023 21.20.19)
2023-02-02 21:20:21.7814 | DEBUG | AudioSource(Spotify) | Using RefreshToken from previous auth to connect to Spotify.
2023-02-02 21:20:22.0127 | DEBUG | AudioSource(Spotify) | Received new access token. Expires in: 01:00:00 (At 02/02/2023 22.20.22)
2023-02-02 22:20:25.0010 | DEBUG | AudioSource(Spotify) | Using RefreshToken from previous auth to connect to Spotify.
2023-02-02 22:20:25.2300 | DEBUG | AudioSource(Spotify) | Received new access token. Expires in: 01:00:00 (At 02/02/2023 23.20.25)
```

**Comment by TechSupport26** [comment]
Can you double-check that there is no mouse binding like this one.
![image](https://user-images.githubusercontent.com/65631811/216604557-1fb7ba2c-4aec-4c25-bdb6-3d41b0e22006.png)

**Comment by sorsted** [comment]
None:
![image](https://user-images.githubusercontent.com/50265631/216604743-e9f9de18-b38f-4bf4-a704-3216e63f01e5.png)

**Comment by TechSupport26** [comment]
The bottom one

**Comment by sorsted** [comment]
As you can see, I am not using any such key binding.

**Comment by TechSupport26** [comment]
LeftClick is the key binding. When you click on AudioBand it takes that as the Mouseinputtype and changes to the NextProfile
You have 4 "Mouse bindings" the bottom one is LeftClick

**Comment by sorsted** [comment]
You are right, that solved the issue. The strange thing is, I am 100 % positive that I never added any of these key bindings. If I had, I would have noticed. So I guess the data must have become corrupted somehow.

That’s why I thought these were predefined key bindings that would only get activated if I checked any of the boxes to the left. So when you asked, I answered that none of them were active. I hope you understand my confusion. 😁
